blog/2024/deep-dive-into-amm-integration.md

```diff
@@ -0,0 +1,114 @@
+
+| Phase                      | Date             |
+|:---------------------------|:-----------------|
+| [AMM Proposal](https://dev.to/ripplexdev/a-proposal-for-the-future-of-the-xrp-ledger-dex-4l7e) | Jul. 2022 |
+| [AMM Performance Testing](https://dev.to/ripplexdev/amm-performance-testing-report-3hon) | Oct. 2023 |
+| [XLS-30 Proposal Voting](https://xrpscan.com/amendments) | Jan. - Mar. 2024 |
+| [XLS-30 Amendment Enabled](https://xrpl.org/resources/known-amendments#amm) | Mar. 2024 |
+| [AMM Bug Fix Integration](https://dev.to/ripplexdev/xrp-ledger-amm-bug-fix-now-integrated-a-detailed-analysis-4915) | Apr. 2024 |
+
+This [consensus](https://xrpl.org/consensus.html) process ensures that changes to the ledger continue to be appropriately scrutinized and vetted before rejection or approval by the community, reflecting a collective agreement on the network’s direction. For any amendment to pass, it must secure at least 80% approval from validators over a two-week continuous voting period, a testament to the XRP Ledger’s commitment to decentralized governance.
+
+To lower the barrier to entry for developers and encourage innovation, several key client libraries ([xrpl.js](https://js.xrpl.org/), [xrpl-py](https://xrpl-py.readthedocs.io/en/stable/), and [xrpl4j](https://xrpl.org/blog/2021/introducing-xrpl4j/)) have been updated and maintained. These libraries have been equipped with functionalities specific to AMM operations, such as creating liquidity pools, managing LP tokens, and executing AMM-specific transactions. Moreover, the [XRPL Explorer](https://amm-devnet.xrpl.org/) has been enhanced to support visibility into AMM pools, offering insights into liquidity depth, transaction history, and a great deal more. 
+
+## What is an Automated Market Maker?
+
+The AMM model on the XRPL is designed to facilitate automatic trading and liquidity provision without the need for traditional order books. This is achieved through the creation of liquidity pools for pairs of assets, where the price of assets is determined algorithmically based on the ratio of the assets in the pool. This model is crucial for ensuring constant liquidity and more stable prices, particularly for pairs that might not have a large volume of direct market trades.
+
+## Core Features of the AMM Integration on XRPL
+
+**Protocol Native:**
+
+* As a core primitive of the XRPL, developers can utilize AMM functionality without the need to create their own smart contracts and face associated risks.
+
+**Aggregated Liquidity:**
+
+* Liquidity for all trading pairs is aggregated at the protocol layer, rather than fragmented or siloed across individual smart contracts. 
+
+**Continuous Auction Mechanism:**
+
+* The implementation of a continuous auction mechanism allows arbitrageurs to bid to capture price discrepancies at a discount.
```

Comment:
_arbitrageurs_ is an acceptable alternate, but _arbitragers_ is fine and more American Engish.